spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -818,6 +818,12 @@
         "level": "error"
       }
     },
+    {
+      "name": "liquibase.test-rollback-on-update",
+      "type": "java.lang.Boolean",
+      "description": "Enable test rollback before starting migration.",
+      "defaultValue": false
+    },
```

Comment:
That section shouldn't be there. The keys around this one are there when we moved `liquibase` to `spring.liquibase` (look at the derepcation). Keys introduced after that migration do not need any warning.